src/app/loadout/fashion/FashionDrawer.tsx

```diff
@@ -515,28 +533,38 @@ function FashionSocket({
         socket?.plugSet?.plugs.some((p) => p.plugDef.hash === plugHash)) ||
       socket?.reusablePlugItems?.some((p) => p.plugItemHash === plugHash && p.enabled));
 
-  return (
-    <ClosableContainer
-      onClose={plugHash ? () => onRemovePlug(bucketHash, plugHash) : undefined}
-      showCloseIconOnHover
-    >
-      {plug ? (
-        <PlugDef
-          onClick={handleOrnamentClick}
-          className={clsx({ [styles.missingItem]: !canSlotOrnament })}
-          plug={(plug ?? defaultPlug) as PluggableInventoryItemDefinition}
-        />
-      ) : (
+  let content = null;
+
+  if (plug) {
+    content = (
+      <PlugDef
+        onClick={handleOrnamentClick}
+        className={clsx({ [styles.missingItem]: !canSlotOrnament })}
+        plug={(plug ?? defaultPlug) as PluggableInventoryItemDefinition}
+      />
+    );
+  } else {
+    content = (
+      <PressTip tooltip={<FashSocketTooltip bucketHash={bucketHash} socket={socket} />}>
         <div
-          role="button"
-          className={clsx('item', { [styles.missingItem]: !canSlotOrnament })}
-          onClick={handleOrnamentClick}
-          tabIndex={0}
-          title={t('FashionDrawer.NoPreference')}
+          role={socket && 'button'}
+          className={clsx('item', {
+            [styles.missingItem]: !canSlotOrnament,
+            [styles.noSocket]: !socket,
+          })}
         >
           <DefItemIcon itemDef={defaultPlug} />
         </div>
-      )}
+      </PressTip>
+    );
+  }
+
+  return (
+    <ClosableContainer
+      onClose={plugHash ? () => onRemovePlug(bucketHash, plugHash) : undefined}
+      showCloseIconOnHover={true}
```

Comment:
Seems like we changed style preference on explicit true boolean props.

```suggestion
      showCloseIconOnHover
```